ui/base-tool/src/main/kotlin/org/cru/godtools/base/tool/activity/BaseToolActivity.kt

```diff
@@ -214,22 +219,23 @@ abstract class BaseToolActivity<B : ViewDataBinding>(@LayoutRes contentLayoutId:
     private fun buildShareIntent(
         title: String? = shareLinkTitle,
         @StringRes message: Int = shareLinkMessageRes,
-        shareUrl: String? = shareLinkUri
+        shareUrl: String? = shareLinkUri,
     ) = shareUrl?.let {
         Intent(Intent.ACTION_SEND).apply {
             type = "text/plain"
             putExtra(Intent.EXTRA_SUBJECT, title.orEmpty())
             putExtra(Intent.EXTRA_TEXT, getString(message, shareUrl))
+            putExtra("EXTRA_SHARE_TOOL_URL", shareUrl)
```

Comment:
I don't think this is necessary anymore now that we are passing the shareUrl via the `DefaultShareItem` constructor